variable/include/scipp/variable/string.h

```diff
@@ -25,6 +25,10 @@ SCIPP_VARIABLE_EXPORT std::ostream &operator<<(std::ostream &os,
 
 SCIPP_VARIABLE_EXPORT std::string to_string(const Variable &variable);
 SCIPP_VARIABLE_EXPORT std::string to_string(const VariableConstView &variable);
+SCIPP_VARIABLE_EXPORT std::string
+to_string(const std::pair<Dim, VariableConstView> &coord);
+SCIPP_VARIABLE_EXPORT std::string
+to_string(const std::pair<std::string, VariableConstView> &attr);
```

Comment:
`attr` is probably a leftover from copy-paste, should be `&vcoord` but completely benign and unimportant

Reply:
This is on purpose: For coords it should hit the overload in line 29 (key is `Dim`). For attributes (and masks) the key is `std::string`.